cybox/test/__init__.py

```diff
@@ -14,6 +17,89 @@
 logger = logging.getLogger(__name__)
 
 
+@contextlib.contextmanager
+def isolation():
+    """Context manager that attempts to reset the sys.modules and globals()
+    to its initial, pre-`import cybox` state.
+
+    This helps eliminate issues where tests pass because of residuals caused
+    by previous imports.
+
+    Any code which leverages this context manager will need to re-import all
+    dependencies within.
+    """
+    def del_sys_modules(to_del):
```

Comment:
Can we move these functions to the top level? I don't like having functions inside functions unless they are necessary for a closure, decorator, or similar.